### PR TITLE
Block unsupported file extensions in the packaging tab

### PR DESCRIPTION
**Pull request by MyunghyunNero**
## Description
- #881
Show a pop-up message when an unsupported file extension is uploaded in the Packaging tab.
Supported file extensions: System > Code Management > Code No: 120 > Code Detail: 16

### TO BE
- English 
스크린샷 2023-07-25 오후 8 22 19

- Korea
스크린샷 2023-07-25 오후 8 33 59

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by 70825 — COMMENTED** [review]
If you modify the content below, it should be fine to approve 👍

**Review by 70825 — APPROVED** [review]
LGTM 👍

File: src/main/java/oss/fosslight/controller/VerificationController.java [hunk 1]
```diff
@@ -72,6 +75,8 @@ public class VerificationController extends CoTopComponent {
 	@Autowired HistoryService historyService;
 	
 	@Autowired VerificationMapper verificationMapper;
+	@Autowired
+	CodeMapper codeMapper;
```
[the remark this answers is not in the document]
Comment (MyunghyunNero) [reply src/main/java/oss/fosslight/controller/VerificationController.java]: Thank you! i modified it😀

**Review by jongwooo — APPROVED** [review]
@MyunghyunNero LGTM. Thank you for your contribution!

**Pull request merged**